tests/integrations/client/http_client_test.go

```diff
@@ -27,6 +27,7 @@ import (
 	pd "github.com/tikv/pd/client/http"
 	"github.com/tikv/pd/client/retry"
 	"github.com/tikv/pd/pkg/core"
+	config2 "github.com/tikv/pd/pkg/schedule/config"
```

Comment:
It seems that other codes use this name
```suggestion
	sc "github.com/tikv/pd/pkg/schedule/config"
```